macos/FluentUITestViewControllers/TestColorViewController.swift

```diff
@@ -20,17 +20,28 @@ class TestColorViewController: NSViewController {
 		colorsStackView.alignment = .leading
 		
 		for color in Colors.Palette.allCases {
-			let colorView = ColorRectView(color: color.color)
-			let textView = NSTextField(labelWithString: color.name)
-			textView.font = .systemFont(ofSize: 18)
-			let rowStackView = NSStackView()
-			rowStackView.orientation = .horizontal
-			rowStackView.spacing = 20.0
-			rowStackView.addArrangedSubview(colorView)
-			rowStackView.addArrangedSubview(textView)
-			colorsStackView.addArrangedSubview(rowStackView)
+			colorsStackView.addArrangedSubview(createColorRowStackView(name: color.name, color: color.color))
 		}
 		
+		/// Excel primary colors
```

Comment:
if we are showcasing excel colors, it should be similar to ios which you can switch between communication blue to excel green.

Reply:
iOS uses a table view to switch between themes, we might have to do the same to achieve that.

Reply:
well instead of creating a table view, couldn't you listen to when the toggle value get changes, unload the entire colorsstackview and re-create it?

Reply:
I see. Maybe we can toggle themes between default and primary.